#! /bin/sh /usr/share/dpatch/dpatch-run
## -*- diff -*-
## fix-comma-in-configure.dpatch by Rafael Laboissiere <rafael@debian.org>
## 
## DP: Fix syntax errors in the configure script

@DPATCH@

--- octave3.1-3.1.50.orig/aclocal.m4
+++ octave3.1-3.1.50/aclocal.m4
@@ -1068,10 +1068,10 @@
 have_opengl_incs=no
 AC_CHECK_HEADERS(GL/gl.h OpenGL/gl.h, [
   AC_CHECK_HEADERS(GL/glu.h OpenGL/glu.h, [
-    have_opengl_incs=yes, break], [], [
+    have_opengl_incs=yes; break], [], [
 #ifdef HAVE_WINDOWS_H
 # include <windows.h>
-#endif ]), break], [], [
+#endif ]) break], [], [
 #ifdef HAVE_WINDOWS_H
 # include <windows.h>
 #endif])
